ReactWindows/ReactNative.Tests/Internal/MockJavaScriptExecutor.cs

```diff
@@ -1,56 +0,0 @@
-﻿using Newtonsoft.Json.Linq;
-using ReactNative.Bridge;
-using System;
-
-namespace ReactNative.Tests
-{
-    class MockJavaScriptExecutor : IJavaScriptExecutor
```

Comment:
>MockJavaScriptExecutor [](start = 10, length = 22)

You may have to leave this if you are going to bring back the JavaScriptModuleBaseTests that were available to UWP before.